pylint/checkers/stdlib.py

```diff
@@ -640,9 +640,17 @@ def _check_open_encoded(self, node: nodes.Call, open_module: str) -> None:
                     encoding_arg = utils.get_argument_from_call(
                         node, position=0, keyword="encoding"
                     )
+                elif open_module == "pathlib" and node.func.attrname == "write_text":
+                    encoding_arg = utils.get_argument_from_call(
```

Comment:
Shall we merge these `if` for `path lib` and then do the `attrname` one. I'm trying to count `if`-calls and its getting late but I think we can reduce the number checks needed to get into L648 from 3 to 2 if you understand what I mean 😅